### PR TITLE
Created atbash_cipher.java file and Updated the README file

### PR DESCRIPTION
**Pull request by SnehaR26**
**Fixes issue:**

**Changes:**

1. Added a java implementation of atbash cipher encryption.
2. Updated the readme file for the same.

### CODE REVIEW

File: code/cryptography/src/atbash_cipher/atbash_cipher.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import java.util.Scanner;
```
Comment (abdouskamel) [inline code/cryptography/src/atbash_cipher/atbash_cipher.java]: Be sure to look at the java style guide, and refactor your code to make it more readable.

File: code/cryptography/src/atbash_cipher/README.md [hunk 2]
```diff
@@ -1,2 +1,24 @@
...
+# Atbash Cipher
+Atbash cipher is a method of encrypting messages. It is a type of monoalphabetic cipher where every letter in the original message (also known as plaintext) is mapped to it's reverse. The first letter becomes the last, the second letter becomes the second last and so on. In the latin alphabet the mapping would be as follows.
+
+| Plain  | A | B | C | D | E | F | G | H | I | J | K | L | M | N | O | P | Q | R | S | T | U | V | W | X | Y | Z |
+| ------ |---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|
+| Cipher | Z | Y | X | W | V | U | T | S | R | Q | P | O | N | M | L | K | J | I | H | G | F | E | D | C | B | A |
+
+_We only try to implement atbash cipher encryption algorithms for latin alphabet._
+
+#### Algorithm
```
Comment (abdouskamel) [inline code/cryptography/src/atbash_cipher/README.md]: You should also talk about decipher.

File: code/cryptography/src/atbash_cipher/atbash_cipher.java (hunk 1 above)
Comment (SnehaR26) [reply code/cryptography/src/atbash_cipher/atbash_cipher.java]: Sure! I'll do that. Thanks for suggestions.

File: code/cryptography/src/atbash_cipher/README.md (hunk 2 above)
Comment (SnehaR26) [reply code/cryptography/src/atbash_cipher/README.md]: I'll add that as well.

File: code/cryptography/src/atbash_cipher/atbash_cipher.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+public class atbash_cipher {
+	public static void main( String[] args) {
+        Scanner scan = new Scanner(System.in);
```
Comment (abdouskamel) [inline code/cryptography/src/atbash_cipher/atbash_cipher.java]: You should indent the body on this function.

File: code/cryptography/src/atbash_cipher/atbash_cipher.java (hunk 1 above)
Comment (abdouskamel) [reply code/cryptography/src/atbash_cipher/atbash_cipher.java]: You should add empty lines between the different parts of your code.

File: code/cryptography/src/atbash_cipher/atbash_cipher.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+			Scanner scan = new Scanner(System.in);
+		
+			String newStr,s;
+		 	char alpha;
+		
+			s = scan.nextLine();
+		
+			newStr = s;
+		
+		// Converting string to character array
```
Comment (abdouskamel) [inline code/cryptography/src/atbash_cipher/atbash_cipher.java]: Comment should be indented the same as instructions.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**